gokart/info.py

```diff
@@ -25,16 +25,27 @@ def make_tree_info(task, indent='', last=True, details=False):
         indent += '|  '
     name = task.__class__.__name__
     result += f'({is_complete}) {name}[{task.make_unique_id()}]'
+
+    if compress:
+        task_list = [] if not task_list else task_list
```

Comment:
Plz use set data structure :)

Reply:
[tips]

```
task_list = task_list or {}
```

is the shorthand pattern for this.

Reply:
🆒